code/Test_definitions/customer-insights-retrieveScoring.feature

```diff
@@ -7,7 +7,7 @@ Feature: CAMARA Customer Insights API, v0.1 - Operation retrieveScoring
   # Testing assets:
   # * A phone number whose scoring can be obtained
   #
-  # References to OAS spec schemas refer to schemas specifies in customer-insights.yaml, version 0.1.0
+  # References to OAS spec schemas refer to schemas specifies in customer-insights.yaml, version 0.1.1
```

Comment:
Here you can remove the version to avoid the duplication, but instead update line 1 with the complete version number v0.1.1

```suggestion
  # References to OAS spec schemas refer to schemas specifies in customer-insights.yaml
```